mods/default/functions.lua

```diff
@@ -327,31 +327,25 @@ end
 
 -- Leafdecay ABM
 
-minetest.register_abm({
-	label = "Leaf decay",
-	nodenames = {"group:leafdecay"},
-	neighbors = {"air"},
-	interval = 2,
-	chance = 10,
-	catch_up = false,
 
-	action = function(pos, node, _, _)
-		-- Check if leaf is placed
-		if node.param2 ~= 0 then
-			return
+function default.search_leaves_for_decay(pos, radius, lradius, trunk_node, leave_node)
```

Comment:
Should be "group:fruits". And fruits should drop and not decay, right? Why do they drop with normal ABM?